src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -1093,6 +1093,7 @@ def _build_record(data):
 def import_zone(cmd, resource_group_name, zone_name, file_name):
     from azure.cli.core.util import read_file_content
     import sys
+    dns_api_version = get_api_version(cmd.cli_ctx, ResourceType.MGMT_NETWORK_DNS)
```

Comment:
get_api_version call is not needed here, since you are using supported_api_version